temporalio/worker/_workflow_instance.py

```diff
@@ -1767,6 +1768,25 @@ def call_later(
         self._pending_timers[seq] = handle
         return handle
 
+    def call_at(
+        self,
+        when: float,
+        callback: Callable[..., Any],
+        *args: Any,
+        context: Optional[contextvars.Context] = None,
+    ) -> asyncio.TimerHandle:
+        # We usually would not support fixed-future-time call (and we didn't
+        # previously), but 3.11 added asyncio.timeout which uses it and 3.12
+        # changed wait_for to use asyncio.timeout. We now simply count on users
+        # to only add to loop.time() and not an actual fixed point. Due to the
+        # fact that loop.time() is at a nanosecond level which floats can't
```

Comment:
I think something like `workflow.time()` would be more obvious for users. Loop time seems like a bit on an unintuitive concept.

Reply:
That already exists. I just use `loop.time()` in this comment and in some other places when discussing this because that's how asyncio refers to it in their samples.

Reply:
OK, cool, I think the README should reference that then rather than loop

Reply:
:+1: Updated